07_errors/kasunfdo/error.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "fmt"
+
+const (
+	ErrInvalidVal = 400
+	ErrNotFound   = 404
+	ErrInternal   = 500
+	ErrOpenDB     = 501
+)
+
+var ErrStr = map[int]string{
+	ErrInvalidVal: "value cannot be negative",
```

Comment:
Too specific. Should be "invalid input" + custom message when you create the error.

Reply:
Changed